nn/normalization.py

```diff
@@ -183,3 +184,44 @@ def normalize(a: nn.Tensor, *, axis: Union[nn.Dim, Sequence[nn.Dim]], epsilon: f
 
   mean, variance = nn.moments(a, axis=axis)
   return (a - mean) * nn.rsqrt(variance + epsilon)
+
+
+class Normalize(nn.Module):
+  """
+  Wrapper around normalize with additional scale and bias
```

Comment:
You should refer to the function correctly.
```suggestion
  Wrapper around :func:`normalize` with additional scale and bias
```

Reply:
But also, I think describing this as "wrapper" is confusing.
Just say:
```suggestion
  :func:`normalize` with additional scale and bias
```